gdeep/trainer/trainer.py

```diff
@@ -514,16 +683,23 @@ def _inner_loop(
         pred_list = []
         batch_metric_list = []
         loss = 0.0
+        ddp_loss = torch.zeros(3).to(self.device)
         with torch.no_grad():
             for X, y in dl:
                 pred, X, y = self._send_to_device(X, y)
                 pred_list.append(pred)
                 class_probs_batch = [f.softmax(el, dim=0) for el in pred]
                 class_probs.append(class_probs_batch)
-                loss += self.loss_fn(pred, y).item()
+                loss = self.loss_fn(pred, y).item()
```

Comment:
@yorickbrunet why did you remove the += here? this "loss" variable is still used to compute the epoch loss a few lines down below (line 704) and I think it may give wrong results without the "+=". Have you checked?

Reply:
This is a mistake. It will be solved in the next push.